ui/src/app/store/machine/types/base.ts

```diff
@@ -166,6 +166,7 @@ export type MachineStatus = {
   unlocking: boolean;
   unlinkingSubnet: boolean;
   unmountingSpecial: boolean;
+  untagging: boolean;
```

Comment:
Do you think it would be useful to use discriminating unions or a similar pattern in such situations (I imagine machine cannot be 'tagging' and 'untagging' at the same time?).

Reply:
Actually a machine totally *could* be "tagging" and "untagging". It's a bit of a misnomer, but these values only indicate that an API request has been made but hasn't resolved yet (something I'm hoping is clearer in my Redux state shape redesign PR)